python-template/{{cookiecutter.placeholder_repo_name}}/requirements/pip-tools.txt

```diff
@@ -4,9 +4,10 @@
 #
 #    make upgrade
 #
-click==7.1.2              # via pip-tools
-pip-tools==5.4.0          # via -r python-template/{{cookiecutter.placeholder_repo_name}}/requirements/pip-tools.in
-six==1.15.0               # via pip-tools
+click==7.1.2
+    # via pip-tools
+pip-tools==5.5.0
```

Comment:
@jmbowman Do you know why comments(# via pip-tools) is starting to show up on new line?

Reply:
That's new in pip-tools 5.5.0: https://github.com/jazzband/pip-tools/releases/tag/5.5.0

Reply:
Yeah, that just started raising a few hackles over in the PR thread now that the release is out: https://github.com/jazzband/pip-tools/pull/1237#issuecomment-753984309 .  I'll be surprised if it gets reverted, but some more heads up and/or an opt-in approach would have been nice.